tools/interactive/interactivetool_wallace.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="interactive_tool_wallace" tool_type="interactive" name="Wallace" version="0.1">
+    <description>Webbased Interactive modeling of species niches and distributions</description>
+    <requirements>
+        <container type="docker">quay.io/bgruening/wilson-app</container>
```

Comment:
@yvanlebras we need the container here.

Reply:
you made it (If I'm not wrong) (how that I am too slow ;) )